edge-hub/core/src/Microsoft.Azure.Devices.Edge.Hub.Core/Constants.cs

```diff
@@ -26,6 +26,7 @@ public class Constants
         public const string SecurityMessageIoTHubInterfaceId = "urn:azureiot:Security:SecurityAgent:1";
 
         public const string ServiceApiIdHeaderKey = "x-ms-edge-moduleId";
+        public const string OriginEdgeHeaderKey = "x-ms-edge-origin";
```

Comment:
is this used only for nested scenario? and is this always edge device id?  if so, we should rename it to make it obvious.

Reply:
any suggestions for naming?

Reply:
if it is used for nested scenario only, should it be "x-ms-edge-nested-origin". @varunpuranik what do you think?